tests/test_fips.py

```diff
@@ -131,7 +149,7 @@ def openssl_fips_hashes_test_fnct(container_per_test: ContainerData) -> None:
 
 
 @pytest.mark.parametrize(
-    "container_per_test", CONTAINERS_WITH_ZYPPER, indirect=True
+    "container_per_test", CONTAINER_IMAGES, indirect=True
```

Comment:
This test only makes sense for containers with openssl installed (this is more or less those with zypper installed). So you need to apply the markers in the same way as you did for `test_openssl_binary`.